modules/dxp/apps/commerce-demo-pack/commerce-theme-minium/commerce-theme-minium-full-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/full/site/initializer/internal/dependencies/journal_articles/MINIUM_PROMO.ftl

```diff
@@ -10,18 +12,26 @@
 
 		window.location.href = Liferay.ctaURL;
 	}
+
+	ctaLink.addEventListener('click', getRelativeURL);
 </script>
 
+<style ${nonceAttribute}>
+	.minium-promo-background-image{
+		background-image: linear-gradient(180deg, rgba(47, 51, 59, 0) 0%, #2F333B 100%), url(${ContentImage.getData()}); border-radius: 8px;
+	}
+</style>
+
 <div class="banner-content-preview">
-	<div class="banner-content-section" style="background-image: linear-gradient(180deg, rgba(47, 51, 59, 0) 0%, #2F333B 100%), url(${ContentImage.getData()}); border-radius: 8px;">
+	<div class="banner-content-section minium-promo-background-image">
 		<div class="banner-content-container">
 			<h1 class="content-preview-title minium-h1">
 				<#if (Heading.getData())??>
 					${Heading.getData()}
 				</#if>
 			</h1>
 
-			<a class="minium-cta-link" onclick="getRelativeURL()">
+			<a class="minium-cta-link">
```

Comment:
If we keep the JS handling, convert to a `button` and try adding also the bootstrap classes for `btn` and `btn-link`. An `a` tag without `href` is invalid HTML. 

You can keep the JS handling if there's no better way to handle this, but could you check that we can simply use the `getLayoutRelativeURL` in a different way? First off, why do we need the relative URL?

If it's not necessary and we can use some other valid URL extracted from `themeDisplay` to attain the same result, maybe we can erase che `<script />` altogether, and convert back to a simple `<a href="...">...</a>`.